### ISSUE TITLE
Refresh token and cookie

### ISSUE THREAD

**Issue by greenkey-m** [markitosgv/JWTRefreshTokenBundle#203]
Lexik_jwt config allow use cookie as token extractor:

    # check token in a cookie
    cookie:
      enabled: true
      name: BEARER

So, is it possible to config refresh token to a cookie? Or it must be coded?
https://github.com/markitosgv/JWTRefreshTokenBundle/issues/132#issuecomment-461333191

**Comment by greenkey-m** [comment]
with a path to refresh,
Set-Cookie: <cookie-name>=<cookie-value>; Path=<path-value>

**Comment by pdugas** [comment]
Working for me... YMMV

```
# config/services.yaml
services:
    app.refreshservice:
        class: App\Service\RefreshService
        public: true
        arguments: [ "@gesdinet.jwtrefreshtoken" ]
```

```
# config/routes.yaml
app_refresh:
    path: /refresh
    methods: [POST]
    #controller: gesdinet.jwtrefreshtoken::refresh
    controller: app.refreshservice::refresh
```

```
# src/Service/RefreshService.php
<?php
namespace App\Service;

use Symfony\Component\HttpFoundation\Request;
use Gesdinet\JWTRefreshTokenBundle\Service\RefreshToken;
use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;

class RefreshService
{
    /** @var RefreshToken */
    private $refreshToken;

    public function __construct(RefreshToken $refreshToken) {
        $this->refreshToken = $refreshToken;
    }

    public function refresh(Request $request)
    {
        if ($bearer = $request->cookies->get('BEARER')) {
            $request->request->set('refresh_token', $bearer);
        }
        return $this->refreshToken->refresh($request);
    }
}
```

**Comment by Jayfrown** [comment]
@greenkey-m I think this can be closed as #274 has been merged, making the requested behavior possible

see: https://github.com/markitosgv/JWTRefreshTokenBundle#configure-refresh-token-in-cookie
